ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2

```diff
@@ -242,17 +242,17 @@ host_monitoring_cron:
 
 {% if osohm_host_type == 'master' %}
 {# Openshift Master checks #}
-- name: run create app every 5 minutes
-  minute: "*/5"
-  job: "ops-runner -f -t 180 -s 60 -n csca.openshift.master.app.create /usr/bin/cron-send-create-app -v --basename pull --source openshift/hello-openshift:v1.0.6 &>> /var/log/create_app.log"
+- name: run create app every 10 minutes
+  minute: "*/10"
+  job: "ops-runner -f -t 480 -s 60 -n csca.openshift.master.app.create /usr/bin/cron-send-create-app -v --basename pull --source openshift/hello-openshift:v1.0.6 &>> /var/log/create_app.log"
```

Comment:
optional: I would include the loop count in this one also - not necessary but helps explain the reason for the new timeout.